lib/awful/titlebar.lua

```diff
@@ -256,6 +260,9 @@ function titlebar.widget.button(c, name, selector, action)
     -- connect to the corresponding signal here.
     c:connect_signal("focus", update)
     c:connect_signal("unfocus", update)
+    c:connect_signal("unmanage", function()
+        c:disconnect_signal("unfocus", update)
+    end)
```

Comment:
This is dead code. The C code unfocuses the client before it emits the unmanage signal, so this can never occur.